clients/android/app/src/main/res/menu/menu_list_files.xml

```diff
@@ -32,12 +32,12 @@
         </menu>
     </item>
 
-    <item
-        android:id="@+id/menu_list_files_search"
-        android:icon="@drawable/ic_baseline_search_24"
-        android:orderInCategory="1"
-        android:title="@string/menu_search"
-        app:showAsAction="always" />
+<!--    <item-->
```

Comment:
what's going on here?

Reply:
I just commented out the search button because I forgot to do it in past merge requests. It was just a button and didn't have anything linked to it.